README.md

```diff
@@ -51,7 +51,12 @@ By typing `docker-compose up` in terminal, you will get:
 - geth eth1 node with private POA network
 - deposit contract deployed on that private eth1 network
     - contract is always deployed on address `0x2F1598e74b146F5687174C13f8EDCF490B2492e3`
-- prysmatic eth2 private network
+- lighthouse eth2 private network
```

Comment:
Add instructions for `docker-compose`

Reply:
Also for creating account
1. importing private key 
2. skip withdrawal
3. configure beacon node: Network localhost, node localhost:5052